packages/botonic-core/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.18.1",
   "license": "MIT",
   "description": "Build Chatbots using React",
-  "main": "src/index.js",
+  "main": "lib/index.js",
```

Comment:
this was something we also talked with @asastre , removing atm and let's see how we coordinate this.

Reply:
Yes, before doing this change we should evaluate all possible implications.
It would be nice too to test if using lib instead of src improve the execution times of the tests, maybe the fact that the code need to be transpiled before the tests is what makes the tests take so long the first time to run.